image-customization.lua

```diff
@@ -22,6 +22,7 @@ packages{
     'ffac-autoupdater-wifi-fallback',
     'ffac-change-autoupdater',
     'ffac-ssid-changer',
+    'ffbs-collect-debug-info'
```

Comment:
```suggestion
    'ffbs-collect-debug-info',
```

Reply:
upsi resolved, thanks grische

Reply:
can we run a linter or something like black? to notice this silly mistakes? 
@grische

Reply:
You can try to add this linter here (not the whole file):
https://github.com/freifunk-gluon/gluon/blob/c5c993582b0e302b6c694e0bda74dc7735c30337/.github/workflows/lint.yml#L20